File: LOO 2°Parcial/Simulador de Vehiculos CSharp/Simulador de Vehiculos CSharp/ValorInvalidoException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Simulador_de_Vehiculos_CSharp
{
    class ValorInvalidoException : Exception
    {
        public ValorInvalidoException() : base("\nValor no valido para modificar la velocidad, por favor ingrese un valor positivo.")
        {

        }
        public String toString()
        {
            return base.ToString();
        }
    }
}

File: LOO 2°Parcial/Simulador de Vehiculos Java/src/Clases/ValorInvalidoException.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package Clases;

/**
 * 
 * @author Dimitri Medina Novelo
 */
public class ValorInvalidoException extends Exception{
    
    public ValorInvalidoException(){
      super("\nValor no valido para modificar la velocidad, por favor ingrese un valor positivo.");
    }
    
    @Override
    public String toString(){
      return super.toString();
    }
}